design-system/react/src/components/InputAmount/InputAmount.tsx

```diff
@@ -26,6 +31,8 @@ export type InputAmountProps = Omit<InputProps, 'size'> & {
   hiddenBalance?: boolean;
   inputProps?: InputNumberProps;
   isDisabled?: boolean;
+  asset?: Asset;
```

Comment:
can we add other optional prop `assetTooltip?: string`?

then if user informs `assetTooltip` we put the tooltip in the asset button